docs/content/latest/benchmark/tpcc-ysql/100-wh/tpcc-ysql.md

```diff
@@ -1,7 +1,9 @@
 ### TPC-C Load Phase
 
 Before starting the workload, you will need to load the data first. Make sure
-to replace the IP addresses with that of the nodes in the cluster.
+to replace the IP addresses with that of the nodes in the cluster. Loader
```

Comment:
Let's add some information in each of these benchmark experiments on:
* What the specs of the benchmark machine were
* How many loader threads were used
* How long it took to load data
* How long the experiment was run for

We can follow this up with a note which says:

Tune the `--loaderthreads` parameter, which determines the concurrency of loading data, depending on the number of nodes and type of nodes in the cluster. The value specified here, 48 threads, is optimal for a 3-node cluster of type `c5d.4xlarge` (16 vCPUs). For larger clusters, or machines with more vCPUs, increase this value accordingly.

Reply:
Done.

Reply:
I didn't have the exact data set size for all the experiments.
Will add them as a subsequent code review when we add the C5d details.